package.json

```diff
@@ -67,9 +67,7 @@
     "history": "^4.9.0",
     "jest-dom": "^3.0.0"
   },
-  "availityWorkflow": {
-    "plugin": "@availity/workflow-plugin-react"
-  },
+  "availityWorkflow": true,
```

Comment:
I think we should also update the `@availity/workflow` dependency and remove `@availity/workflow-plugin-react`. And we should eventually get all the regular dependencies updated